install.sh

```diff
@@ -291,13 +290,16 @@ nvm_do_install() {
   fi
 
   # Sourcing $PROFILE to take into account the new environment
-  . "$NVM_PROFILE"
+  . "$NVM_DIR/nvm.sh"
 
   nvm_check_global_modules
 
   nvm_install_node
 
   nvm_reset
+
+  echo "=> Close and reopen your terminal to start using nvm or run the following to use it now:"
+  echo "=> export NVM_DIR=\"$NVM_DIR\" && . \"\$NVM_DIR/nvm.sh\""
```

Comment:
can this use `$SOURCE_STR` so that it's always the same as the lines added to the profile?